### DISCUSSION TITLE
Hey! Thanks for this project.

### DISCUSSION THREAD

**Discussion by shymega** [microsoft/windows-drivers-rs#9] [General]
Hi!

I've been looking at generating my own Windows driver bindings for my TDP driver - https://github.com/shymega/WinLinTDPControl - but this hits the spot.

I can't wait to see this project develop, and seeing as my target device of handhelds is mostly Windows, this really paves the way.

Thank you :)

**Comment by wmmc88** [comment]
Thanks for your interest in this project! We're still early on in this project, so let us know if you find any bugs or have ideas on improving it. We always welcome feedback!
